src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -57,6 +57,7 @@
    :master-run-dir (schema/maybe schema/Str)
    :master-log-dir (schema/maybe schema/Str)
    (schema/optional-key :track-lookups) schema/Bool
+   (schema/optional-key :disable_i18n) schema/Bool
```

Comment:
Does this have to be optional to support upgrades? Or could we require it and just set its default in `initialize-puppet-config` above, like we do with all the other keys? Same question for `track-lookups`.

Reply:
I don't know about any requirements for this to be optional to support upgrades. My interpretation here was that those values set in `initialize-puppet-config` are required to successfully start puppet, but I don't know if that interpretation is correct.